feature-libs/cart/base/styles/components/_added-to-cart-toast.scss

```diff
@@ -0,0 +1,101 @@
+:root {
+  --cx-added-to-cart-toast-width: 375px;
+  --cx-added-to-cart-toast-header-offset: 192px;
+  --cx-added-to-cart-toast-background: var(--cx-color-inverse);
+  --cx-added-to-cart-toast-translateY: -100%;
+}
+
+cx-added-to-cart-toast {
```

Comment:
Many component style definition start with a `%`, would that apply here?

Reply:
Actually, now I know that this class should start with a % to make it an extensible sass abstract class.  Please implemente it as such.